src/main/java/de/adorsys/keycloak/config/service/ClientImportService.java

```diff
@@ -353,4 +362,50 @@ private void removeAuthorizationPolicy(String realm, ClientRepresentation client
             // policies got deleted if linked resources are deleted, too.
         }
     }
+
+    private void updateClientAuthenticationFlowBindingOverrides(RealmImport realmImport, List<ClientRepresentation> clients) {
+        String realm = realmImport.getRealm();
+
+        for (ClientRepresentation client : clients) {
+            ClientRepresentation existingClient = clientRepository.getClientByClientId(realm, client.getClientId());
+            updateAuthenticationFlowBindingOverrides(realm, existingClient, client.getAuthenticationFlowBindingOverrides());
+        }
+    }
+
+    private void updateAuthenticationFlowBindingOverrides(String realm, ClientRepresentation client, Map<String, String> authenticationFlowBindingOverrides) {
+        Map<String, String> authFlowUpdates = null;
+
+        // Be sure that all existing values will be cleared
+        if (client.getAuthenticationFlowBindingOverrides() != null) {
+            authFlowUpdates = client.getAuthenticationFlowBindingOverrides().entrySet()
+                    .stream()
+                    .collect(Collectors.toMap(
+                            Map.Entry::getKey,
+                            e -> ""
+                    ));
+        }
```

Comment:
Is this required? 

You could set `authFlowUpdates` to an empty Map in line 375 and remove the condition in line 401+402 to just the else case.

Or does Keycloak require that all entries have to be an empty string?

Reply:
Sadly; Keycloak require that all entries have to be an empty string in order to remove them

Reply:
Ouhh... 

Could you has this Sadly information as commentary inside code? I will forget this in a few weeks... Thanks!